daemon/algod/api/algod.oas2.json

```diff
@@ -1096,6 +1096,65 @@
         }
       }
     },
+    "/v2/transactions/simulate": {
+      "post": {
+        "consumes": [
+          "application/x-binary"
```

Comment:
As far as I can tell, the format does not affect our server-side code generation in any way.

Perhaps we decided `application/x-binary` is more accurate, since group transactions are encoded as concatenated msgpack objects?

@winder, do you have more insight into this?

Reply:
I think this is just for documentation. My research at the time suggested that `application/x-msgpack` was practically an unofficial standard, but we ended up going with something slightly more technically official.

Reply:
Thanks for thoughts. I suppose I agree it's not _really_ msgpack because we just encode consecutive txns.